vms/platformvm/txs/mempool/mempool.go

```diff
@@ -154,52 +153,62 @@ func (m *mempool) DisableAdding() {
 	m.dropIncoming = true
 }
 
-func (m *mempool) Add(tx *txs.Tx) error {
+func (m *mempool) Add(txs []*txs.Tx) error {
```

Comment:
This won't add any of the later txs in `txs` if one in the middle fails a check. I think we should retain the single tx `Add`

Reply:
Thanks for spotting this. Fixed while keeping the addition of a tx slice. I really like the simmetry with Remove